src/index.js

```diff
@@ -1,7 +1,47 @@
+const client = require('prom-client');
 const config = require('config');
 const fetchStatuCakeData = require('./statuscake');
 const server = require('./server');
+const Storage = require('./statuscakedata');
 
-this.interval = setInterval(fetchStatuCakeData, config.statuscake.interval);
+const regex = /\s?([-])\s?/;
+
+const getTagsFromTitle = Title => Title.split(regex);
+
+const g = new client.Gauge({
```

Comment:
I hate single letter variable 😅 
Could you name it with something meaningful? 🙏

Reply:
Done 🙆‍♂